source/as-promise.ts

```diff
@@ -56,7 +56,7 @@ export default function asPromise(options: NormalizedOptions): CancelableRequest
 				return;
 			}
 
-			if (response.req?.aborted) {
+			if (response.req.aborted) {
```

Comment:
```suggestion
			if (response.req?.aborted) {
```

Because the cached response doesn't have `req` property

Reply:
Okies - will reflect this change in the `Response` type too!

Reply:
Done :)